src/java/org/apache/cassandra/tools/LoaderOptions.java

```diff
@@ -60,10 +72,14 @@
     public static final String IGNORE_NODES_OPTION = "ignore";
     public static final String CONNECTIONS_PER_HOST = "connections-per-host";
     public static final String CONFIG_PATH = "conf-path";
-    public static final String THROTTLE_MBITS = "throttle";
-    public static final String INTER_DC_THROTTLE_MBITS = "inter-dc-throttle";
-    public static final String ENTIRE_SSTABLE_THROTTLE_MBITS = "entire-sstable-throttle";
-    public static final String ENTIRE_SSTABLE_INTER_DC_THROTTLE_MBITS = "entire-sstable-inter-dc-throttle";
+    @Deprecated
+    public static final String THROTTLE_MEGABITS = "throttle";
```

Comment:
Good call! 
Cassandra is not provided as a library, but people do use it for the purpose. Compatibility is best effort. Those constants can be perfectly as "private", but declared as "public". See https://issues.apache.org/jira/browse/CASSANDRA-10637
I think the changes of the constants names are not strictly required. We can deprecate the original names and add a comment saying "MBITS" are "MEGAMITS". 
WDYT? @frankgh

Reply:
I think it's best to keep the original name, since Cassandra could sometimes be used as a library. I will add the comment and I will retain the `@Deprecated` annotation as well.

Reply:
Thanks for sharing that ticket @yifan-c , I think we can even add it to the comments for other people like us tempted to make changes here :-)